controller/blogy.js

```diff
@@ -0,0 +1,62 @@
+const Blog = require("../model/post");
+const Comment = require("../model/comment");
+const asyncWrapper = require('../middleware/async')
+
+
+// For blog post
+const createSinglePost = asyncWrapper( async (req, res) => {
+    const { title, content, post } = req.body;
+    const newBlog = new Blog({ title, content, post });
+    await res.json({ newBlog })
```

Comment:
1. You do not `await` res.json.
2. res.json should be the last line in this function
3.

Reply:
Now changed.